src/system/WatchableEventManagerLibevent.cpp

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
Derived from `WatchableSocketLibevent.cpp`; see https://gist.github.com/kpschoedel/7e7b22d4ea056d9f37e46bae88112d53 for diff